examples/warm-start-qaoa.yaml

```diff
@@ -176,9 +176,18 @@ steps:
   - cost_function_specs:
       module_name: zquantum.core.cost_function
       function_name: AnsatzBasedCostFunction
-      estimator-specs:
-        module_name: zquantum.core.estimator
-        function_name: ExactEstimator
+      estimation_method_specs:
+        module_name: zquantum.core.estimation
+        function_name: calculate_exact_expectation_values
+      estimation_preprocessors_specs:
+      - grouping: # This string is just for syntax and informative purposes
+        module_name: zquantum.core.estimation 
+        function_name: group_greedily_with_context_selection
+      - shot_allocation: # This string is just for syntax and informative purposes
+        module_name: zquantum.core.estimation 
+        function_name: allocate_shots_uniformly
+        number_of_shots: 10000
```

Comment:
`grouping` and `shot_allocation` should be removed. This yaml converts to the following json:

```
{
  "grouping": null,
  "module_name": zquantum.core.estimation,
  ...
}
```